### DISCUSSION TITLE
Libs not working after the recent update

### DISCUSSION THREAD

**Discussion by AlexSkykid070** [thatskymod/Sky-CotL-Scripts#75] [General]
please fix..

**Comment by thatskymod** [comment]
Duplicate of #69, #72 and #74 and resolved in Dukun Langit v0.2.3rc:

https://github.com/thatskymod/Sky-CotL-Scripts/blob/d021c69403acb5cc6f9d705c510bc3f110e47bb7/Canvas/Libs/libLangit023rc.so
